dask_planner/src/sql.rs

```diff
@@ -108,11 +108,52 @@ impl ContextProvider for DaskSQLContext {
             let rtf: ReturnTypeFunction = Arc::new(|_| Ok(Arc::new(DataType::Int64)));
             return Some(Arc::new(ScalarUDF::new("year", &sig, &rtf, &fun)));
         }
+        if "atan2".eq(name) {
+            let sig = Signature::variadic(
+                vec![DataType::Float64, DataType::Float64],
+                Volatility::Immutable,
+            );
+            let rtf: ReturnTypeFunction = Arc::new(|_| Ok(Arc::new(DataType::Float64)));
+            return Some(Arc::new(ScalarUDF::new("atan2", &sig, &rtf, &fun)));
+        }
+        if "cbrt".eq(name) {
+            let sig = Signature::variadic(vec![DataType::Float64], Volatility::Immutable);
+            let rtf: ReturnTypeFunction = Arc::new(|_| Ok(Arc::new(DataType::Float64)));
+            return Some(Arc::new(ScalarUDF::new("cbrt", &sig, &rtf, &fun)));
+        }
```

Comment:
For some methods with the same signature and return types it might make sense to consolidate into a single if arm with or conditions. Something like if `"atan2".eq(name) | "cot".eq(name) ...`

Reply:
Makes sense, thanks! I'll update that.